dts/common/yaml/zephyr-i2c.yaml

```diff
@@ -0,0 +1,19 @@
+---
+title: Zephyr specific I2C properties
+id: i2c
+version: 0.1
+
+description: >
+    This binding gives Zephyr specific I2C properties
+
+properties:
+  - zephyr,speed :
+      type: int
+      category: required
+      description: Speed setting (1..5) as defined in i2c.h
+      generation: define
+  - zephyr,use-10-bit-addr :
```

Comment:
I've opened a Jira issue related to this problem: https://jira.zephyrproject.org/browse/ZEP-2373